sds_data_manager/utils/stackbuilder.py

```diff
@@ -161,7 +161,7 @@ def build_sds(
         scope=sdc_stack,
         construct_id="RDS",
         vpc=networking.vpc,
-        engine_version=rds.PostgresEngineVersion.VER_15_6,
+        engine_version=rds.PostgresEngineVersion.VER_15_7,
```

Comment:
Is there a reason to only bump this one minor version rather than go to the latest (do we fail with higher versions)? Otherwise, I think we are going to be in the same situation in another couple of months.